uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.controller.js

```diff
@@ -66,6 +68,7 @@ define([
     vm.statusFieldVisible = statusFieldVisible;
     vm.showStatusField = showStatusField;
     vm.searchContactAssignments = searchContactAssignments;
+    vm.validateFileSize = validateFileSize;
```

Comment:
Why is it public? I assume for testing purpose? Is there any other way around to avoid `vm.` ?

Reply:
Yes! for testing purpose. if we make the `validateFileSize` private  the function won't be testable. I welcome any suggestions though.

Reply:
Sure. You may try and change tests so they check that documents are not uploaded if their size exceed limits (spy uploading function). If this function is an internal function, it should not be revealed to the outer interface.

Reply:
ok! l will that way then. thanks